### DISCUSSION TITLE
Why can't my "kenv's script" be triggered through shortcut keys?

### DISCUSSION THREAD

**Discussion by alkene5** [johnlindquist/kit#1288] [Q&A] [answered]
<img width="768" alt="image" src="https://github.com/johnlindquist/kit/assets/61105068/966fe30b-2a9e-4ccf-b203-0337a3d4259c">

**Comment by johnlindquist** [comment]
@alkene0005 That looks like a Chinese keyboard layout translation issue. Are you trying `cmd+shift+k` or `cmd+shift+l`?

**Reply by alkene5** [reply]
I use the English input method is the same
When my script is in the ~/.kenv/scripts directory, it can be triggered by the shortcut key
However, if my script is in the ~/.kenv/kenvs/xxx/scripts directory, it cannot be triggered by the shortcut key

**Reply by johnlindquist** [reply, marked as answer]
@alkene0005 Oh, that means you'll need to "trust" the kenv to allow shortcuts and other "automatic" script running:

![CleanShot 2023-06-27 at 17 01 44](https://github.com/johnlindquist/kit/assets/36073/9cf91ad4-21a2-4350-bcfa-fa03167ed092)

**Reply by alkene5** [reply]
I think I solved it, I've tried it before and it didn't work, but it worked this time, which is a little weird, but it should work out
